frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -88,23 +89,35 @@ pods:
           rmi-port:
             port: {{SECURE_JMX_RMI_PORT}}
           {{/TASKCFG_ALL_SECURE_JMX_ENABLED}}
+          {{#PROMETHEUS_EXPORTER_ENABLED}}
+          exporter:
+            port: 9500
+          {{/PROMETHEUS_EXPORTER_ENABLED}}
       sidecar-resources:
         cpus: 1
         memory: 1024
     tasks:
       server:
         goal: RUNNING
         resource-set: server-resources
+        {{#PROMETHEUS_EXPORTER_ENABLED}}
+        labels: "DCOS_METRICS_FORMAT:prometheus,DCOS_METRICS_PORT_NAME:exporter,DCOS_METRICS_ENDPOINT:/metrics"
+        {{/PROMETHEUS_EXPORTER_ENABLED}}
         cmd: |
                 set -e
 
                 export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
                 ./bootstrap
                 ARGS='-f'
 
-                if [ -n "$STATSD_UDP_HOST" ]; then
-                    ARGS="$ARGS -Dcassandra.metricsReporterConfigFile=metrics-reporter-config.yaml"
+                {{#PROMETHEUS_EXPORTER_ENABLED}}
+                echo 'JVM_OPTS="$JVM_OPTS -javaagent:$MESOS_SANDBOX/cassandra-exporter-agent-0.9.10.jar={{PROMETHEUS_EXPORTER_CONFIGURATION_OPTIONS}}"' >> apache-cassandra-{{CASSANDRA_VERSION}}/conf/cassandra-env.sh
```

Comment:
can we avoid hardcoded version of exporter here with introducing some {{EXPORTER_VERSION}} variable?

Reply:
@alexeygorobets I was thinking to bring version.sh in Cassandra as even though I will provide {{EXPORTER_VERSION}} in svc.yml (by providing version in marathon.json) but in resource.json hardcoded version will still be there!, so I was thinking to bring all the version in version.sh in separate PR, it this sounds good to you ?

Reply:
@alexeygorobets is that ok ?

Reply:
Hi, @viivek46 sure, let's do it in a separate PR